python/Produce_Config_File.py

```diff
@@ -0,0 +1,118 @@
+import xlrd
+from array import array
+import numpy as np
+import ROOT as r
+from options import parser
+import sys
+
+if len(sys.argv)==1:
+    parser.print_help()
+    sys.exit(1)
+    pass
+
+def Color(j):
+	if j % 7 == 0:
+		ret_int='kRed'
+		pass
+	elif j % 7 == 1:
+		ret_int='kRed+1'
+		pass
+	elif j % 7 == 2:
+		ret_int='kRed+2'
+		pass
+	elif j % 7 == 3:
+		ret_int='kRed+3'
+		pass
+	elif j % 7 == 4:
+		ret_int='kBlue'
+		pass
+	elif j % 7 == 5:
+		ret_int='kBlue+1'
+		pass
+	elif j % 7 == 6:
+		ret_int='kBlue+2'
+		pass
+	return ret_int
+
+args = parser.parse_args()
+text_file = open(str(args.OutputName)+".cfg", "w")
```

Comment:
Isn't this is already a string, why are you casting it again as a string?  This is unnecessary